packages/inferno-server/src/renderToString.queuestream.ts

```diff
@@ -251,21 +241,25 @@ export class RenderQueueStream extends Readable {
 					this.addToQueue(renderedString + html + '</' + type + '>', position);
 					return;
 				}
-				// Close element if it's not void
 				if (!isVoidElement) {
 					this.addToQueue(renderedString + '</' + type + '>', position);
 				}
 			}
-			// Push text directly to queue
 		} else if (flags & VNodeFlags.Text) {
-			this.addToQueue((firstChild ? '' : '<!---->') + escapeText(children), position);
-			// Handle errors
+			this.addToQueue(
+				(firstChild ? '' : '<!---->') + escapeText(children),
+				position
+			);
 		} else {
 			if (process.env.NODE_ENV !== 'production') {
 				if (typeof vNode === 'object') {
-					throwError(`renderToString() received an object that's not a valid VNode, you should stringify it first. Object: "${ JSON.stringify(vNode) }".`);
+					throwError(
```

Comment:
It indents the template literal on the next line. It's way past 80 columns so there's not much more prettier can do.